### DISCUSSION TITLE
saving Amelia's parameters and applying to new data

### DISCUSSION THREAD

**Discussion by jeandigitale** [IQSS/Amelia#46] [Q&A]
Hi,

I'm interested in using Amelia to impute in an training dataset and then applying the parameters from the algorithm to a held-out test set as part of a larger prediction pipeline. I see in the notes of the CRAN PDF: "Note that the theta, mu and covMatrcies objects refers to the data as seen by the EM algorithm and is thusly centered, scaled, stacked, tranformed and rearranged. See the manual for details and how to access this information." Is there a way to apply the same manipulation to my held-out test set so that I could impute based on the EM algorithm from my training set? 

Thanks,
Jean
